Detectors/ITSMFT/ITS/macros/test/CheckCOG.C

```diff
@@ -106,7 +107,17 @@ void CheckCOG(std::string clusfile = "o2clus_its.root", std::string inputGeom =
 
       if (dict.IsGroup(cComp.getPatternID())) {
         if (patternsPtr) { // Restore the full pixel pattern information from the auxiliary branch
-          auto patt = (*patternsPtr)[nc];
+          auto rowSpan = *pattIdx++;
+          auto colSpan = *pattIdx++;
+
+          int nBytes = rowSpan * colSpan / 8;
+          if (((rowSpan * colSpan) % 8) != 0)
+            nBytes++;
+          unsigned char tmp[Cluster::kMaxPatternBytes];
+          for (int i = 0; i < nBytes; i++)
+            tmp[i] = *pattIdx++;
+
+          o2::itsmft::ClusterPattern patt(rowSpan, colSpan, tmp);
```

Comment:
do we need this? Is not it better to make getClusterCoordinates operate directly over the pattIdx using the lightweight  ClusterPattern helper we discussed?

Reply:
@shahor02  Right. I will have a look.  If it does not break anything anywhere around, I will try to add this change to this PR.